tools/testsys/src/aws_ecs.rs

```diff
@@ -193,6 +193,12 @@ impl CrdCreator for AwsEcsCreator {
         Ok(CreateCrdOutput::NewCrd(Box::new(Crd::Test(test_crd))))
     }
 
+    async fn workload_crd<'a>(&self, _test_input: TestInput<'a>) -> Result<CreateCrdOutput> {
+        Err(error::Error::Invalid {
+            what: "Workload testing is not supported for non-k8s variants".to_string(),
```

Comment:
Just curious, will adding ECS be a follow on from this?

Reply:
Yes. Ecs workload testing will be available after TestSys 0.0.6 is out.